src/porepy/numerics/nonlinear/nonlinear_solvers.py

```diff
@@ -138,8 +138,8 @@ def newton_step() -> None:
                 ):
                     solver_progressbar.set_description_str(
                         "Newton iteration number "
-                        + f"{model.nonlinear_solver_statistics.num_iteration + 1} of \t"
-                        + f"{self.params['max_iterations']}"
+                        + f"{model.nonlinear_solver_statistics.num_iteration + 1} of"
+                        + f" {self.params['max_iterations']}"
```

Comment:
```suggestion
                        + f" {self.params['max_iterations']}."
```

Reply:
The progress bar adds a colon automatically :)